deployment/updates/scripts/xml/2019_04_28_tagEqualsBooleanNotification.template.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i dont see deployment script for this template

Reply:
The deployment already existed. Just the XML didn't... Someone deleted the XML file between Quasar-17.2.0 and Quasar-17.3.0. I am just bringing it back.